packages/web-config-server/src/apiV1/dataBuilders/helpers/calculateOperationForAnalytics.js

```diff
@@ -24,8 +30,14 @@ const valueToGroup = (value, config) => {
 };
 
 const performSingleAnalyticOperation = (analytics, config) => {
-  const { operator, dataElement } = config;
-  const filteredAnalytics = analytics.filter(({ dataElement: de }) => de === dataElement);
+  const { operator } = config;
+  // filterOptions could be ['dataElement', 'organisationUnit'] for multiple matching key options
+  const filterOptions = config.filterOptions ?? config.dataElement;
+  const filterOptionsValues = {};
+  filterOptions.forEach(filterOptionKey => {
+    filterOptionsValues[filterOptionKey] = config[filterOptionKey];
+  });
```

Comment:
`Object.fromEntries` is often handy when creating value maps:

```suggestion
const valuesByFilterKey = Object.fromEntries(filterOptions.map(key => [key, config[key]]));
```